uisdk/src/main/res/drawable/uisdk_gradient_enabled.xml

```diff
@@ -2,7 +2,7 @@
     android:shape="rectangle">
     <gradient
         android:angle="0"
-        android:endColor="@color/primary"
-        android:startColor="@color/primary"
+        android:endColor="@color/secondary"
```

Comment:
I suggest using a similar approach as the previous one`KhButtonDisabledColor`

Reply:
Done